src/utils.ts

```diff
@@ -55,11 +56,18 @@ export async function createFiles(rootDir: string, files: Map<string, string>, f
   }
 }
 
-export function determinePackageManager(rootDir: string): 'yarn' | 'npm' {
+export function determinePackageManager(rootDir: string): PackageManager {
```

Comment:
I wonder if this could be improved - we know that respective manager is running this code, can we look at `process.argv` and determine it based on what's currently running?

Reply:
That's a good idea. However, I'm not sure how you want to handle the `process.env.npm_config_user_agent` logic. My intention was only to add support for `pnpm`, not to refactor the existing code. I would suggest to handle the refactoring outside of the scope of this PR to keep changes isolated. Not sure what are your team workflow practices.

Reply:
We do drive-by fixes in the code that we touch. I saw you adding another `fs.existsSync` and that triggered the question. After taking another look it sounds like all modern managers are setting the environment variable, so we can rely on that and drop the fs.existsSync in lines 60 and 62, and a remove check in line 64, falling back to npm. Does that sound reasonable?